skimage/color/colorconv.py

```diff
@@ -41,7 +41,7 @@
 :author: Travis Oliphant (XYZ and RGB CIE functions)
 :author: Matt Terry (lab2lch)
 :author: Alex Izvorski (yuv2rgb, rgb2yuv and related)
-
```

Comment:
I think generally we are now adding each person's contributions to the file `CONTRIBUTORS.txt`.

@stefanv should Imram add his name here too for this module?